packages/wallets/src/helpers/urls.ts

```diff
@@ -155,3 +157,10 @@ export const redirectToOutSystems = (currency = '') => {
         return (window.location.href = redirectURL.toString());
     }
 };
+
+export const redirectToStandaloneP2P = () => {
+    const baseUrl = isProduction() ? derivUrls.DERIV_P2P_PRODUCTION : derivUrls.DERIV_P2P_STAGING;
+
+    const redirectURL = new URL(baseUrl);
+    return (window.location.href = redirectURL.toString());
```

Comment:
Actually why not just 👀 
```suggestion
    window.location.href = redirectURL.toString());
```

Reply:
Updated 👍